src/api-guides/e-receipts.markdown

```diff
@@ -1,173 +1,159 @@
-- To ensure that you are using the correct endpoint URLs, the safest practice is to check the service index before every request.   [Get service Index](/api-reference/receipts/endpoints.html#endpoint-service-index)
-- If your app currently generates a receipt image, (.png, .jpg,.tif, .pdf or other image file), you can include it in your post to Concur.
-- Concur generates a standard receipt for Partners who do not tender a receipt image.
-- Concur will display your App Logo on standard generated receipts (size 100x100 px)
+- To ensure that you are using the correct endpoint URLs, the safest practice is to check the service index before every request. [Get service Index](/api-reference/receipts/endpoints.html#endpoint-service-index)
+- If your app currently generates a receipt image, (.png, .jpg, .tif, .pdf, or other image file), you can include it in your post to SAP Concur.
+- SAP Concur generates a standard receipt for Partners who do not tender a receipt image.
+- SAP Concur will display your App Logo on standard generated receipts (size 100x100 px).
 
-#### Post an E-Receipt after payment has been made
+### Post an E-Receipt After Payment Has Been Made
 
 1. Obtain a fresh access token for the user using [refresh grant](/api-reference/authentication/apidoc.html#refresh_token).
-2. You will need the user **Id** previously stored in the user&#39;s profile to post receipts on his/her behalf.
-3. Post receipt using the [Post Receipt](/api-reference/receipts/endpoints.html#endpoint-post-a-receipt) endpoint
-4. You should store the Concur\_Correlation\_ID and the Location Link from the response headers even for successful responses, this will facilitate support efforts in case of a missing receipt from User&#39;s Concur account.
+2. You will need the user `Id` previously stored in the user's profile to post receipts on his/her behalf.
+3. Post receipt using the [Post Receipt](/api-reference/receipts/endpoints.html#endpoint-post-a-receipt) endpoint.
+4. You should store the Concur\_Correlation\_ID and the Location Link from the response headers even for successful responses, this will facilitate support efforts in case of a missing receipt from User's SAP Concur account.
 
+## Token Management
 
-### Token Management
-
-#### Refreshing Tokens
+### Refreshing Tokens
 
 1. You will obtain a fresh accesstoken before making API calls on behalf of your users using the [refresh grant](/api-reference/authentication/apidoc.html#refresh_token).
-2. A user&#39;s refreshtoken is valid for six months.
-3. The below elements are returned with the new Access Token, replace previously stored values for the user in your database:
-   - **refresh\_token** _(Replace)_
-   - **refresh\_expires\_in** _(Replace)_
-4. If the user does not log in for six months his/her refresh token will expire and the user&#39;s Partner account and Concur account will be de-linked.
+2. A user's `refreshToken` is valid for six months.
```

Comment:
Technically speaking, it is the `refreshToken` for the application, not the user. You can avoid the confusion altogether by saying 'A `refreshToken` is valid for six months.`